rental.rb

```diff
@@ -0,0 +1,26 @@
+require './book'
+require './person'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(book, person, date)
+    @book = book
+    @person = person
+    @date = date
+
+    book.add_rental(self)
+    person.add_rental(self)
```

Comment:
- [ ] Add_rental method should take different parameters for each class(person, book). So you should use
 `book.rentals << self` 
`person.rentals << self` instead this block.